### PR TITLE
LPS-88710

### PR DESCRIPTION
**Pull request by yuhai**
Hi @tinatian,

The PR is based on https://github.com/tinatian/liferay-portal/pull/1154, no logic changes and only for solving the conflict with the latest master.

cc @dantewang

Thanks,
Hai

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:test:relevant

File: modules/apps/portal-store/portal-store-s3/src/main/java/com/liferay/portal/store/s3/AbortedMultipartUploadCleaner.java [hunk 1]
```diff
@@ -66,9 +66,12 @@ protected void activate(BundleContext bundleContext)
 			bundleContext.createFilter(
 				StringBundler.concat(
 					"(&(", Constants.OBJECTCLASS, "=", Store.class.getName(),
-					")(store.type=", S3Store.class.getName(), "))")),
+					")(store.type=", S3Store.class.getName(),
+					")(current.store=true))")),
```
Comment (tinatian) [inline modules/apps/portal-store/portal-store-s3/src/main/java/com/liferay/portal/store/s3/AbortedMultipartUploadCleaner.java:70]: @yuhai @dantewang 
Is this change related to scheduler? If not, let's create a new ticket and make the change.

Also, is this a bug? if yes, please provide more info in the comment and steps to reproduce it in the new ticket.

Thanks.
Tina

**Comment by tinatian** [comment]
@yuhai @dantewang 
Please list what you test in details.

Thanks.
Tina.

File: modules/apps/portal-store/portal-store-s3/src/main/java/com/liferay/portal/store/s3/AbortedMultipartUploadCleaner.java (hunk 1 above)
Comment (dantewang) [reply modules/apps/portal-store/portal-store-s3/src/main/java/com/liferay/portal/store/s3/AbortedMultipartUploadCleaner.java:70]: @tinatian This is a bug in the AbortedMultipartUploadCleaner where 
1. the service tracker is never opened;
2. the tracked service maybe added twice

Will resend without this commit, and address it with details in another ticket.

**Comment by yuhai** [comment]
please see https://github.com/tinatian/liferay-portal/pull/1165
